packages/zosfiles/src/cli/hMigrate/HMigrate.definintion.ts

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Please check the file name. `HMigrate.definintion.ts` ==> `HMigrate.definition.ts`